### DISCUSSION TITLE
Issues with Twitter API Keys

### DISCUSSION THREAD

**Discussion by tommo593** [mindsdb/mindsdb#6261] [Get help]
Hi,

I am trying to create a bot like Snoopstein. I am having issues with my Twitter API keys as when I run the query it says "Can't connect to db: 403 Forbidden

When authenticating requests to the Twitter API v2 endpoints, you must use keys and tokens from a Twitter developer App that is attached to a Project. You can create a project via the developer portal."

I have the correct Twitter account and my app is in my project and I'm using the correct keys. What could be the issue?

**Comment by ZoranPandovski** [comment]
We recommend you use the [Elevated access](https://developer.twitter.com/en/support/twitter-api/developer-account) allowing you to pull 2m tweets and to avoid parameters or authentication issue error you might get sometimes. You can check [this step-by-step guide](https://medium.com/@skillcate/set-up-twitter-api-to-pull-2m-tweets-month-44d004c6f7ce) describing how to apply for the Elevated access.

**Reply by tommo593** [reply]
Does the Elevated access not cost money? The next best option I've seen costs $100 a month.

**Reply by ZoranPandovski** [reply]
I just doubled check Elevated: Free access up to 2M Tweets/month, and 3 App environments. 3 Apps, 1 Project. Requires an approved developer account application.

**Reply by tommo593** [reply]
I see, I think my issue might be that my account might not be approved yet. I'll look into it, thanks a lot!
